tests/belongs_to_many_test.js

```diff
@@ -36,7 +36,8 @@ var Support = {
       typeValidation: true,
       define: {
         paranoid: true
-      }
+      },
+      dialectOptions: {cockroachdbTelemetryDisabled : true},
```

Comment:
do you know why only this test has a config override? aren't there a lot of other tests in the `tests/` dir?

Reply:
All the other tests have their tests defined in the test helper. This one uses some special config (not entirely sure why off the top of my head)